web3/_utils/abi.py

```diff
@@ -97,25 +95,26 @@ def filter_by_name(name: str, contract_abi: ABI) -> List[Union[ABIFunction, ABIE
         for abi
         in contract_abi
         if (
-            # type ignored b/c see line 91
-            abi['type'] not in ('fallback', 'constructor') and  # type: ignore
-            abi['name'] == name  # type: ignore
+            abi['type'] not in ('fallback', 'constructor') and
+            abi['name'] == name
         )
     ]
 
 
-def get_abi_input_types(abi: Union[ABIFunction, ABIEvent]) -> List[str]:
+def get_abi_input_types(abi: ABIFunction) -> List[str]:
     if 'inputs' not in abi and abi['type'] == 'fallback':
         return []
     else:
-        return [collapse_if_tuple(arg) for arg in abi['inputs']]
+        # https://github.com/python/mypy/issues/4976
```

Comment:
is this comment outdated?  maybe a text comment with a brief explanation would be good too?